search/search_quality_tests/search_quality_tests.cpp

```diff
@@ -219,7 +219,8 @@ int main(int argc, char * argv[])
   vector<double> responseTimes(queries.size());
   for (size_t i = 0; i < queries.size(); ++i)
   {
-    string const & query = queries[i];
+    // todo(@m) Add a bool flag to search with prefixes?
+    string const & query = queries[i] + " ";
```

Comment:
It works but won't the last token be treated as a prefix then? I don't want it to.